### PR TITLE
COR-658: add office field

### PR DESCRIPTION
**Pull request by internetti**
add an office field for data collection

### CODE REVIEW

File: internal/api/validation/individual.go [hunk 1]
```diff
@@ -84,6 +85,7 @@ func validateIndividual(i *api.Individual, p *validation.Path) validation.ErrorL
 const (
 	individualAddressMaxLength                        = 512
 	individualAdministrativeAreaMaxLength             = 128
+	individualCollectionOfficeMaxLength               = 128
```
Comment (neb42) [inline internal/api/validation/individual.go:88]: Not really a comment about this pr, but how certain are we about these max lengths?

File: cmd/mockdata/main.go [hunk 2]
```diff
@@ -217,6 +218,7 @@ func Generate(count uint) error {
 		collectionAdministrativeArea1 := f.Country()
 		collectionAdministrativeArea2 := f.State()
 		collectionAdministrativeArea3 := f.City()
+		collectionOffice := f.City() + " Office"
```
[the remark this answers is not in the document]
Comment (internetti) [reply cmd/mockdata/main.go:221]: I get why you say that, but this is just producing the fake data, I don't think it matters much, if this is a different city. 
so far, the fake cities are usually not even in the fake countries, so nothing matches here

File: internal/api/validation/individual.go (hunk 1 above)
Comment (internetti) [reply internal/api/validation/individual.go:88]: I personally am not. I just figured the office falls into the same category as the areas (locations), so I went with the same number.

Comment (neb42) [reply internal/api/validation/individual.go:88]: Yeah I think it's the right call based on that, but I think these probably need reviewing

**Comment by neb42** [comment]
lgtm

**Pull request merged**